resources/views/individuals/partials/progress.blade.php

```diff
@@ -23,7 +23,8 @@
         @if ($individual->checkStatus('draft'))
             <p class="stack">
                 <button class="secondary" name="preview" value="1">{{ __('Preview page') }}</button>
-                <button class="secondary" name="publish" value="1">{{ __('Publish page') }}</button>
+                <button class="secondary" name="publish" value="1"
+                    @cannot('publish', $individual) disabled @endcannot>{{ __('Publish page') }}</button>
```

Comment:
I was thinking following our earlier conversation, we'll need to address this but probably should do so in a separate PR to keep this one focused.